### DISCUSSION TITLE
consult：how to switch remote branches

### DISCUSSION THREAD

**Discussion by GiveMe-A-Name** [extrawurst/gitui#1565] [Q&A] [answered]
I don't understand what key is map the icon?
![image](https://user-images.githubusercontent.com/58852732/221353764-793b8706-91d8-454a-a4af-0e8c9dd0e3fb.png)

It seems that README doesn't specify what the icon means, so I don't think the icon is obvious what kind of key is.

**Comment by extrawurst** [comment, marked as answer]
It’s a `tab`
